flink-libraries/flink-table/src/main/scala/org/apache/flink/table/plan/nodes/datastream/DataStreamGroupWindowAggregate.scala

```diff
@@ -107,6 +107,14 @@ class DataStreamGroupWindowAggregate(
     val groupingKeys = grouping.indices.toArray
     val inputDS = input.asInstanceOf[DataStreamRel].translateToPlan(tableEnv)
 
+    val consumeRetraction = DataStreamRetractionRules.isAccRetract(input)
+
+    if (consumeRetraction) {
+      throw new TableException(
+        "Retraction on group window is not supported yet. Note: Currently, group window should " +
+          "not follow an unbounded groupby.")
```

Comment:
1. 'is not supported yet' -> 'unsupported yet' maybe more concisely.
2. groupby -> groupBy